portality/models/account.py

```diff
@@ -246,3 +246,9 @@ def get_name_safe(cls, account_id) -> str:
             if author is not None and author.name:
                 return author.name
         return ''
+
+    @classmethod
+    def is_enable_publisher_email(cls) -> bool:
+        # TODO: in the long run this needs to move out to the user's email preferences but for now it
+        # is here to replicate the behaviour in the code it replaces
+        return app.config.get("ENABLE_PUBLISHER_EMAIL", False)
```

Comment:
I'm not sure that moving this config here currently makes sense, even with your note explaining why.

`ENABLE_PUBLISHER_EMAIL` is currently a **global** setting on the application, so it would make more sense to only register the consumer if `ENABLE_PUBLISHER_EMAIL` is True (i.e. move this to the `should_consume` in portality/events/consumers/update_request_publisher_submitted_notify.py)

However, I agree there _should_ be a user-set flag on their account to disable their own email notifications. In which case, it would go here and have a checkbox + model change to support it

Reply:
since User might have different email preferences in the future 
move function to `Account.is_enable_publisher_email` is **"pre-design"** for that feature.

even `Account.is_enable_publisher_email` using `ENABLE_PUBLISHER_EMAIL` is not make sense for now.
That would be still fine because `Account.is_enable_publisher_email` and `ENABLE_PUBLISHER_EMAIL` are **temporary** until User base email preferences implemented.

 I would suggest move `is_enable_publisher_email` to account_service too when implement that feature to avoid models layer too complex.

Reply:
I accept it for the timebeing :D